lib/stm32/common/flash_common_f24.c

```diff
@@ -24,6 +24,7 @@
 
 /**@{*/
 
+#include <string.h>
```

Comment:
also this is probably not necessary now that memcpy is gone

Reply:
Good catch, thanks